Assets/Scripts/Point.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class Point : MonoBehaviour
+{
+    [SerializeField]
+    private Color _highlightColor = Color.yellow;
+    private Color _startColor;
+    private BasicsCreator.Mode _startMode;
```

Comment:
You're referring to `BasicsCreator` which doesn't seem to be part of this pull request as far as I can tell:
```
Assets/Scripts/Point.cs(11,13): error CS0246: The type or namespace name 'BasicsCreator' could not be found (are you missing a using directive or an assembly reference?)
```